examples/osgviewerIPhone/shaders.h

```diff
@@ -0,0 +1,113 @@
+#pragma once 
```

Comment:
I feel that it would be more consistent to use the osg_Color in the vertex shader and pass this as a varying to fragment shaders rather than use a custom uniform.